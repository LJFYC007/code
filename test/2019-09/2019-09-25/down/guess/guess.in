5
82 71 83 24
78 47 77 36
14 3 7 11
76 1 43 64
48 79 21 100
