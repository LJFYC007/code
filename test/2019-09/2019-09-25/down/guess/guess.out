74 2 2
2 50 32
2 8 4
60 52 32
32 32 16
